app/main.py

```diff
@@ -1,8 +1,51 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
+                 comfort_class: int,
+                 clean_mark: int,
+                 brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
```

Comment:
```suggestion
def __init__(
         self,
         comfort_class: int,
         clean_mark: int,
         brand: str
 ) -> None:
```

Reply:
what's wrong here? I don't see the message